pallets/tokens/src/lib.rs

```diff
@@ -186,6 +193,8 @@ decl_storage! {
 		///
 		/// NOTE: This is only used in the case that this module is used to store balances.
 		pub Accounts get(fn accounts): double_map hasher(blake2_128_concat) T::AccountId, hasher(twox_64_concat) T::CurrencyId => AccountData<T::Balance>;
+
+        NextCurrencyId get(fn next_asset_id): T::CurrencyId;
```

Comment:
maybe once this PR is verified, we can run cargo fmt to fix such indentation errors, before merging.

Reply:
i would not format external crates as it might be hard in the future to upgrade to newer version (because of the conflicts) - for our code i have just applied formatting

Reply:
probably would bo good to have some github check that enforces that :thinking: